server/public/model/config.go

```diff
@@ -900,6 +901,10 @@ func (s *ServiceSettings) SetDefaults(isUpdate bool) {
 	if s.RefreshPostStatsRunTime == nil {
 		s.RefreshPostStatsRunTime = NewString("00:00")
 	}
+
+	if s.MaximumPayloadSize == nil {
+		s.MaximumPayloadSize = NewInt64(100000)
```

Comment:
Can we get a reasonable default from security? Was this arbitrarily chosen or was there a method to it?

Reply:
This value was essentially arbitrary but I have gone through several of the APIs, including ones that pass in an object looking for reasons this number might be limiting. I think this number is probably a bit higher even than necessary but too small risks breaking things. Also the reason for it being configurable.

@esarafianou - Do you have recommendations?

Reply:
> I think this number is probably a bit higher even than necessary but too small risks breaking things. 

Agreed with that. I don't have any other recommendation.